releasenotes/notes/release-train-02-2023-package-refresh-9de8049f8cb08a5e.yaml

```diff
@@ -0,0 +1,18 @@
+---
+features:
+  - |
+    Package repositories and container images for CentOS Stream based deployments have been updated.
+    Key packages to note are:
+
+    * Kernel
+      * version: 4.18.0
+      * release: 408.el8
```

Comment:
Is the kernel version correct? 408 was released in July and already present in the last package sync. I would have expected 448: https://koji.mbox.centos.org/koji/buildinfo?buildID=23410